src/main/java/com/google/api/generator/gapic/composer/ServiceClientSampleCodeComposer.java

```diff
@@ -169,4 +180,140 @@ public static String composeClassHeaderEndpointSampleCode(
             ExprStatement.withExpr(initSettingsVarExpr),
             ExprStatement.withExpr(initClientVarExpr)));
   }
+
+  public static String composeRpcMethodHeaderSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    // TODO(summerji): Add other types RPC methods' sample code.
+    return SampleCodeWriter.write(
+        composeUnaryRpcMethodSampleCode(method, arguments, clientType, resourceNames));
+  }
+
+  public static TryCatchStatement composeUnaryRpcMethodSampleCode(
+      Method method,
+      List<MethodArgument> arguments,
+      TypeNode clientType,
+      Map<String, ResourceName> resourceNames) {
+    VariableExpr clientVarExpr =
+        VariableExpr.withVariable(
+            Variable.builder()
+                .setName(JavaStyle.toLowerCamelCase(clientType.reference().name()))
+                .setType(clientType)
+                .build());
+    // List of rpc method arguments' variable expressions.
+    List<Expr> rpcMethodArgVarExprs =
```

Comment:
The reason I use `List<VarialbeExpr>` here is `rpcMethodArgVarExprs` is used to pass in as `List<Expr>` for `MethodInvocationExpr.setArguments`  in other two places.  See [line262](https://github.com/googleapis/gapic-generator-java/pull/573/files#diff-7f479f381fc848c9556f6d57565536498b3f517034d769e0d36ad4d5136f13f3R262) and [line273](https://github.com/googleapis/gapic-generator-java/pull/573/files#diff-7f479f381fc848c9556f6d57565536498b3f517034d769e0d36ad4d5136f13f3R273).
And L246 is good place to cast as variable expr because it is the place need for-loop on each element. Otherwise, we need create another for-loop to cast expr to variable expr somewhere.

Reply:
Good point. In this case, let's use `List<VariableExpr>` and remove the cast, then in MethodArgument, use a stream to cast back up to Expr. The reason is that casting wider is safer - `(Expr) VariableExpr` will always be OK, but `(VariableExpr) Expr` is not, since the `Expr` could be anything else.

Reply:
Make sense.